.github/workflows/report-code-coverage.yml

```diff
@@ -41,11 +42,17 @@ jobs:
         uses: codecov/codecov-action@v3.1.4
         with:
           directory: ./reports/coverage
+      
+      - name: Upload Coverage Reports to Codecov
```

Comment:
```suggestion
      - name: Upload TS Coverage Reports to Codecov
```

Reply:
Also, did this work? I mean, did Codecov aggregate all the coverage data?

Reply:
Step title updated and yes, Codecov aggregated all the coverage data